amd/src/acceptmodal.js

```diff
@@ -0,0 +1,222 @@
+
+
+/**
+ * Add a create new group modal to the page.
+ *
+ * @module     tool_policy/acceptmodal
+ * @class      AcceptOnBehalf
+ * @package    tool_policy
+ * @copyright  2018 Marina Glancy
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+define(['jquery', 'core/str', 'core/modal_factory', 'core/modal_events', 'core/fragment', 'core/ajax', 'core/yui'],
+    function($, Str, ModalFactory, ModalEvents, Fragment, Ajax, Y) {
+
```

Comment:
This is where I usually put
```
"use strict";
```
as I think our own and new modules should be linted and developed with high standards. And having it inside the module's wrap function does not affect evaluating others' code.